### DISCUSSION TITLE
https://theme-hope.vuejs.press/guide/markdown/others.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#561] [Announcements]
# https://theme-hope.vuejs.press/guide/markdown/others.html

Link check The theme will check your Markdown links in dev mode by default. You can customize this feature through checkLinks in plugin options. checkLinks receives an object. Y...

https://theme-hope.vuejs.press/guide/markdown/others.html

**Comment by venkat-Reddy-b** [comment]
Thanks
Very easy to understand and helpful for the beginners like me
